tests/functional/setups/manu_flow_incomp_frac_2d.py

```diff
@@ -755,17 +754,6 @@ class ManuIncompSolutionStrategy2d(
     results: list[ManuIncompSaveData]
     """List of SaveData objects."""
 
-    def __init__(self, params: dict):
-        """Constructor for the class."""
-
-        super().__init__(params)
-
-        self.exact_sol: ManuIncompExactSolution2d
```

Comment:
Just to be sure I understand: Is exact_sol now not declared anywhere?

Reply:
It's declared in the class attribute section, as per (our) mixin convention.